File: Python Code/Daily_Flash_Code_0/Week 3/28Jan/13-DailyFlash_Solutions/13-DailyFlash_Solutions/28_Jan_Solutions_One/Java/Factorial.java
import java.io.*;


class Factorial{
	
	public static void main(String args[])throws IOException{

		BufferedReader br  = new BufferedReader(new InputStreamReader(System.in));

		System.out.printf("Enter the lowerlimit : ");
		long num1 = Long.parseLong(br.readLine());
		System.out.printf("Enter the upperlimit : ");
		long num2 = Long.parseLong(br.readLine());
		long fact = 1;
		for(long  itr = num1 ; itr<=num2 ; itr++){
		
			fact = 1;

			for(long inner = 1 ; inner<=itr ; inner++)
				fact = fact*inner;

			System.out.println("Factorial of "+itr+" is "+fact);		
	

		}
	}	
}		

File: Python Code/Daily_Flash_Code_0/Week 3/28Jan/13-DailyFlash_Solutions/13-DailyFlash_Solutions/28_Jan_Solutions_One/Cpp/factorial.cpp
#include<iostream>

int main(){
	
	int num1,num2;

	printf("Enter the lowerlimit : ");
	scanf("%d",&num1);

	printf("Enter the upperlimit : ");
	scanf("%d",&num2);

	
	for(int  itr = num1 ; itr<=num2 ; itr++){
		
		int fact = 1;

		for(int inner = 1 ; inner<=itr ; inner++)
			fact = fact*inner;

		printf("Factorial of %d is %d\n ",itr,fact);		


	}	

}	
